README.md

```diff
@@ -11,6 +11,11 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   sudo apt-get install python-wstool
   ```
 
+- Install QD either from [source](http://crd-legacy.lbl.gov/~dhbailey/mpdist/) or, on Ubuntu 14.04 or later, through apt-get:
+ ```
+ sudo apt-get install libqd-dev
+ ```
+
```

Comment:
Shouldn't this also be installed by `rosdep install ..`? It is declared a `run` and `build_depend` of `godel_surface_detection` ([here](https://github.com/ros-industrial-consortium/godel/blob/78ee0f4cb757035114f759110e7d5d69159a1b96/godel_surface_detection/package.xml#L15))?